app/controllers/api/samples_controller.rb

```diff
@@ -0,0 +1,40 @@
+class Api::SamplesController < ApplicationController
+  before_action :set_user, only: [:show, :update]
+    
+    def index
```

Comment:
you are apparently using 4 spaces indentation
normally it's just 2, so if you didn't agree internally to use 4 make sure to set your IDE to handle it properly